olp-cpp-sdk-dataservice-read/src/repositories/DataRepository.cpp

```diff
@@ -162,10 +162,25 @@ BlobApi::DataResponse DataRepository::GetBlobData(
     }
   }
 
+  // Check if other threads have faced an error.
+  const auto optional_error = mutex.GetError();
+  if (optional_error) {
+    OLP_SDK_LOG_DEBUG_F(kLogTag,
+                        "An error is stored in NamedMutex, aborting, hrn='%s', "
```

Comment:
This gives the impression that you are storing the error here.
```suggestion
                        "Found error in NamedMutex, aborting, hrn='%s', "
```

Reply:
Done